packages/dev/core/src/Meshes/Compression/dracoCompression.ts

```diff
@@ -1,468 +1,34 @@
-
-                return new AutoReleaseWorkerPool(numWorkers, () => {
-                    return new Promise((resolve, reject) => {
-                        const worker = new Worker(workerBlobUrl);
-                        const onError = (error: ErrorEvent) => {
-                            worker.removeEventListener("error", onError);
-                            worker.removeEventListener("message", onMessage);
-                            reject(error);
-                        };
-
-                        const onMessage = (message: MessageEvent) => {
-                            if (message.data === "done") {
-                                worker.removeEventListener("error", onError);
-                                worker.removeEventListener("message", onMessage);
-                                resolve(worker);
-                            }
-                        };
-
-                        worker.addEventListener("error", onError);
-                        worker.addEventListener("message", onMessage);
-
-                        worker.postMessage({
-                            id: "init",
-                            decoder: {
-                                url: decoderInfo.url,
-                                wasmBinary: decoderWasmBinary,
-                            },
-                        });
-                    });
-                });
-            });
-        } else {
-            this._decoderModulePromise = decoderInfo.wasmBinaryPromise.then((decoderWasmBinary) => {
-                if (!decoderInfo.url) {
-                    throw new Error("Draco decoder module is not available");
-                }
-
-                return Tools.LoadScriptAsync(decoderInfo.url).then(() => {
-                    return createDecoderAsync(decoderWasmBinary as ArrayBuffer);
-                });
-            });
-        }
-    }
-
-    /**
-     * Stop all async operations and release resources.
-     */
-    public dispose(): void {
-        if (this._workerPoolPromise) {
-            this._workerPoolPromise.then((workerPool) => {
-                workerPool.dispose();
-            });
+        if (!DracoCompression._Default0) {
+            DracoCompression._Default0 = new DracoCompression();
         }
 
-        delete this._workerPoolPromise;
-        delete this._decoderModulePromise;
+        return DracoCompression._Default0;
     }
 
-    /**
-     * Returns a promise that resolves when ready. Call this manually to ensure draco compression is ready before use.
-     * @returns a promise that resolves when ready
-     */
-    public whenReadyAsync(): Promise<void> {
-        if (this._workerPoolPromise) {
-            return this._workerPoolPromise.then(() => {});
-        }
-
-        if (this._decoderModulePromise) {
-            return this._decoderModulePromise.then(() => {});
-        }
-
-        return Promise.resolve();
-    }

```

Comment:
This is also a breaking change. Also, I didn't see this method in the encoder/decoder. Assuming we add this to the encoder/decoder, we need to forward this call to the decoder.